src/components/user/repo/trashbin.repo.integration.test.js

```diff
@@ -95,4 +100,154 @@ describe('user repository', () => {
 			expect(result.data.length).to.equal(2);
 		});
 	});
+
+	describe('deleteExpiredData', () => {
+		it('should delete data older than the backupPeriodThreshold', async () => {
```

Comment:
describe: when delete expired data with giving a threshold
it: should _only_ delete data lower than threshold
it: should keep/ignore _all_ other data [which is higher than threshold]